next/app/about/leadership/page.tsx

```diff
@@ -0,0 +1,194 @@
+      },
+      {
+        avatar: "https://randomuser.me/api/portraits/lego/6.jpg",
+        name: "Kaelyn Beeman",
+        title: "Marketing",
+        desc: "Lorem Ipsum is simply dummy text of the printing and typesettin industry.",
+        linkedin: "javascript:void(0)",
+        github: "javascript:void(0)",
+        email: "javascript:void(0)",
+      },
+      {
+        avatar: "https://randomuser.me/api/portraits/lego/7.jpg",
+        name: "Emily Chrisostomo",
+        title: "Student Outreach",
+        desc: "Lorem Ipsum is simply dummy text of the printing and typesettin industry.",
+        linkedin: "javascript:void(0)",
+        github: "javascript:void(0)",
+      },
+      {
+        avatar: "https://randomuser.me/api/portraits/lego/8.jpg",
+        name: "Ryan Webb",
+        title: "Tech",
+        desc: "Lorem Ipsum is simply dummy text of the printing and typesettin industry.",
+        linkedin: "javascript:void(0)",
+        github: "javascript:void(0)",
+      },
+    ],
+  };
+
+  return (
+    <>
+      <section className="mt-16">
+        <div className="max-w-screen-xl mx-auto px-4 text-center md:px-8">
+          <div className="content-center">
+            {/* Meet our team */}
+            <div className="max-w-xl mx-auto">
+              <h1
+                className="bg-gradient-to-t from-primary to-secondary 
+              bg-clip-text text-4xl font-extrabold text-transparent md:text-5xl"
+              >
+                Meet our team
+              </h1>
+              <p className="mt-3 text-xl leading-8">
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit. Lorem,
+                ipsum dolor sit amet consectetur adipisicing elit. Lorem, ipsum
+                dolor sit amet consectetur.
+              </p>
+            </div>
+          </div>
+
+          {/* Primary Officers */}
+          <div className="my-20">
+            <h3 className="text-xl font-extrabold text-primary-focus sm:text-3xl my-12">
+              Primary Officers
+            </h3>
+            <div className="">
+              <div className="grid gap-6 sm:grid-cols-2 md:grid-cols-4">
+                {team.primary_officers.map((item, idx) => (
+                  <MemberCard key={idx} item={item} />
+                ))}
+              </div>
+            </div>
+          </div>
+          {/* Committee Heads */}
+          <div className="mt-20">
+            <h3 className="text-xl font-extrabold text-primary-focus sm:text-3xl mb-12">
+              Committee Heads
+            </h3>
+            <div className="">
+              <div className="grid gap-6 sm:grid-cols-2 md:grid-cols-4">
+                {team.committee_heads.map((item, idx) => (
+                  <MemberCard key={idx} item={item} />
+                ))}
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </>
+  );
+}
+
+const MemberCard = ({ item }) => {
```

Comment:
Let's pull this component into a separate file, to clean up the code in this file. You could create an `OfficerCard.tsx` in the same folder alongside page.tsx, and export the component to be imported in the page.tsx file.

Reply:
✅